Week-3/ex1.js

```diff
@@ -0,0 +1,29 @@
+const memoize = (fn) => {
+  let cache = new Map();
+  return function (...args) {
+    let [a, b] = args;
```

Comment:
@jasham Do we need to destructure args as a, b ? We can directly pass the args right to the function. Reason i am asking is, this can be more than 2 arguments as well. Please correct it

Reply:
Sure @RajeshSivanesan I will correct it.